qiskit/pulse/instructions/acquire.py

```diff
@@ -146,50 +110,54 @@ def acquire(self) -> AcquireChannel:
         """Acquire channel to acquire data. The ``AcquireChannel`` index maps trivially to
         qubit index.
         """
-        return self._acquires[0] if self._acquires else None
+        return self._channel
 
     @property
     def mem_slot(self) -> MemorySlot:
         """The classical memory slot which will store the classified readout result."""
-        return self._mem_slots[0] if self._mem_slots else None
+        return self._mem_slot
 
     @property
     def reg_slot(self) -> RegisterSlot:
         """The fast-access register slot which will store the classified readout result for
         fast-feedback computation.
         """
-        return self._reg_slots[0] if self._reg_slots else None
+        return self._reg_slot
 
     @property
     def acquires(self) -> List[AcquireChannel]:
         """Acquire channels to be acquired on."""
-        return self._acquires
+        warnings.warn("Acquire.acquires is deprecated. Use the channel attribute instead.",
+                      DeprecationWarning)
+        return [self._channel]
 
     @property
     def mem_slots(self) -> List[MemorySlot]:
         """MemorySlots."""
-        return self._mem_slots
+        warnings.warn("Acquire.mem_slots is deprecated. Use the mem_slot attribute instead.",
+                      DeprecationWarning)
+        return [self._mem_slot]
 
     @property
     def reg_slots(self) -> List[RegisterSlot]:
         """RegisterSlots."""
-        return self._reg_slots
+        warnings.warn("Acquire.reg_slots is deprecated. Use the reg_slot attribute instead.",
+                      DeprecationWarning)
+        return [self._reg_slot]
 
     def __repr__(self) -> str:
         return "{}({}{}{}{}{}{})".format(
             self.__class__.__name__,
             self.duration,
-            ', ' + ', '.join(str(ch) for ch in self.acquires) if self.acquires else '',
+            ', ' + str(self.channel) if self.channel else '',
             ', ' + str(self.mem_slot) if self.mem_slot else '',
             ', ' + str(self.reg_slot) if self.reg_slot else '',
             ', ' + str(self.kernel) if self.kernel else '',
             ', ' + str(self.discriminator) if self.discriminator else '')
 
     def __call__(self,
-                 channel: Optional[Union[AcquireChannel, List[AcquireChannel]]] = None,
-                 mem_slot: Optional[Union[MemorySlot, List[MemorySlot]]] = None,
-                 reg_slots: Optional[Union[RegisterSlot, List[RegisterSlot]]] = None,
-                 mem_slots: Optional[Union[List[MemorySlot]]] = None,
+                 channel: AcquireChannel,
```

Comment:
Shouldn't it be same as `__init__` so we warn on deprecated?

```suggestion
                 channel: Optional[AcquireChannel] = None,
```

Reply:
Well, this is the deprecated call feature:
```
# deprecated
acquire = Acquire(200)(acq_channel)
```
We have to allow channel to be none in init to enable the above line. But, we don't have to support this:
```
acquire = Acquire(200)()  # no channel passed??
```

Reply:
However, since it is deprecated, I also don't care much about fixing up its behavior, so if you still think it should match, I'm happy to apply your suggestion.

Reply:
`acquire = Acquire(200)()` brings much more confusion than missmatch of init and call 😄 
After deprecation period is over `channel` in `__init__` will be required parameter, all good

Reply:
ok! Thank you for bringing it up!